### PR TITLE
add probe status checking

### PR DESCRIPTION
**Pull request by ss1h2a3tw**
For issue https://github.com/lxc/lxc/issues/1604
Signed-off-by: Shane Chen <ss1ha3tw@gmail.com>

### CODE REVIEW

File: src/lxc/tools/lxc-checkconfig.in [hunk 1]
```diff
@@ -28,16 +28,27 @@ is_enabled() {
...
+            $SETCOLOR_WARNING && echo -n "missing" && $SETCOLOR_NORMAL
         fi
     fi
+    return $RET
+}
+
+is_probed() {
+    lsmod | grep $1 > /dev/null
+    if [ $? -eq 0 ]; then
+        echo -n " probed"
```
Comment (brauner) [inline src/lxc/tools/lxc-checkconfig.in]: Hm, wouldn't it be better to say: "kernel module loaded" and "kernel module not loaded". "probed" "no probed" might confuse users.

**Comment by ss1h2a3tw** [comment]
maybe "loaded" and "not loaded" will be better? 
printing out full sentence is quite long for me.

**Comment by brauner** [comment]
Do you have sample output that you can paste here?

**Comment by ss1h2a3tw** [comment]
`--- Namespaces ---                                                                                                                                                      
Namespaces: enabled                                                                                                                                                     
Utsname namespace: enabled                                                                                                                                              
Ipc namespace: enabled                                                                                                                                                  
Pid namespace: enabled                                                                                                                                                  
User namespace: enabled                                                                                                                                                 
Network namespace: enabled                                                                                                                                              
                                                                                                                                                                        
--- Control groups ---
Cgroup: enabled                                                                                                                                                         
Cgroup clone_children flag: enabled                                                                                                                                     
Cgroup device: enabled                                                                                                                                                  
Cgroup sched: enabled                                                                                                                                                   
Cgroup cpu account: enabled                                                                                                                                             
Cgroup memory controller: enabled                                                                                                                                       
Cgroup cpuset: enabled                                                                                                                                                  

--- Misc ---                                                                                                                                                            
Veth pair device: enabled kernel module loaded                                                                                                                          
Macvlan: enabled kernel module not loaded                                                                                                                               
Vlan: enabled kernel module not loaded                                                                                                                                  
Bridges: enabled kernel module loaded                                                                                                                                   
Advanced netfilter: enabled kernel module loaded                                                                                                                        
CONFIG_NF_NAT_IPV4: enabled kernel module loaded                                                                                                                        
CONFIG_NF_NAT_IPV6: enabled kernel module not loaded                                                                                                                    
CONFIG_IP_NF_TARGET_MASQUERADE: enabled kernel module loaded                                                                                                            
CONFIG_IP6_NF_TARGET_MASQUERADE: enabled kernel module not loaded                                                                                                       
CONFIG_NETFILTER_XT_TARGET_CHECKSUM: enabled kernel module not loaded                                                                                                   
FUSE (for use with lxcfs): enabled kernel module not loaded                                                                                                             
                                                                                                                                                                        
--- Checkpoint/Restore ---                                                                                                                                              
checkpoint restore: missing                                                                                                                                             
CONFIG_FHANDLE: enabled                                                                                                                                                 
CONFIG_EVENTFD: enabled                                                                                                                                                 
CONFIG_EPOLL: enabled                                                                                                                                                   
CONFIG_UNIX_DIAG: enabled                                                                                                                                               
CONFIG_INET_DIAG: enabled                                                                                                                                               
CONFIG_PACKET_DIAG: enabled                                                                                                                                             
CONFIG_NETLINK_DIAG: enabled                                                                                                                                            
File capabilities: enabled                                                                                                                                              
                                                                                                                                                                        
Note : Before booting a new kernel, you can check its configuration
usage : CONFIG=/path/to/config ./lxc-checkconfig
`

**Comment by brauner** [comment]
Yeah, `loaded`, `not loaded` should be fine but then you should be `enabled, loaded`.

**Comment by ss1h2a3tw** [comment]
ok

**Pull request merged**